production/docker/docker-compose-ha-memberlist.yaml

```diff
@@ -57,32 +24,78 @@ services:
     command: "-config.file=/etc/loki/loki-docker-memberlist-boltdb-shipper.yaml -target=all"
     networks:
       - loki
-    restart: on-failure
+    restart: always
```

Comment:
Do you think it would be good to set restart policy to `unless-stopped` instead of `always`?
I am not used to docker-compose much but I did some digging to understand restart policy and found [this](https://rollout.io/blog/ensuring-containers-are-always-running-with-dockers-restart-policy/) useful post.

Reply:
yes, i'm already update always to on-failure